editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
```

Comment:
Just did it for this go around, so the output looked nicer, when the actual component is finalized it should definitely be grabbing the html.